recipes/xerces-c/bld.bat

```diff
@@ -0,0 +1,29 @@
+
+if %ARCH% == 64 (
+    set PLATF=x64
+    set OUTDIR=Win64
+) else (
+    set PLATF=Win32
+    set OUTDIR=Win32
+)
+
+if %PY3K% == 1 (
```

Comment:
are there VC14 projects?  If not, I recommend the upgrade path shown in https://github.com/conda/conda-build/wiki/Windows-recipe-patterns#updating-older-sln-files

Actually, just updating the VS2008 solution might be cleanest anyway.